modules/apps/notification/notification-test/src/testIntegration/java/com/liferay/notification/service/test/NotificationTemplateLocalServiceTest.java

```diff
@@ -109,6 +118,55 @@ public void testAddNotificationTemplate() throws Exception {
 			notificationTemplate);
 	}
 
+	@Test
+	public void testAddNotificationTemplateError() throws Exception {
+		User user = TestPropsValues.getUser();
+
+		AssertUtils.assertFailure(
```

Comment:
Could you move this logic to the top of `testAddNotificationTemplate` method, please?